app/templates/editor.hbs

```diff
@@ -74,7 +74,7 @@
             title=(readonly post.titleScratch)
             titlePlaceholder="Story Title"
             onTitleChange=(action "updateTitleScratch")
-            onTitleBlur=(action (perform saveTitle))
+            onTitleBlur=(route-action "saveTitle")
```

Comment:
Using route actions like this is deprecated, anything that needs to be done can be done inside the `saveTitle` task.